src/components/main-form.js

```diff
@@ -25,6 +25,7 @@ import { FV, PMT } from 'formulajs/lib/financial'
 import styled, { css, cx } from 'react-emotion'
 import { getYearly } from '../utils/timeSeriesResultsByOption'
 
+import classNames from 'classnames'
```

Comment:
Could you accomplish the same thing w/ https://5bb1495273f2cf57a2cf39cc--emotion.netlify.com/docs/cx?

Reply:
yes, you're right. Just gotten used to classnames, I'll uninstall it.